src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/DetailsViewModelMapper.cs

```diff
@@ -48,6 +53,9 @@ public async Task<DetailsViewModel> Map(DetailsRequest source)
                 (var singleOption, var hasOptions) = await HasOptions(data.Apprenticeship.StandardUId);
                 var showOptions = hasOptions && !preDateStandardVersioning;
 
+                var apiRequest = new GetEditApprenticeshipRequest(source.ProviderId, source.ApprenticeshipId);
```

Comment:
There's a strict 1:1 mapping between UI pages (and their underlying mappers, orchestrators, etc.) and endpoints on the outer api. Since this request is for the _edit_ page, we can't use it here. We'll have to add a new endpoint to apim for this page and use that instead, even to fetch (initially) a single property. (The work involved to do this isn't as much as it sounds and should only take an hour or so).